build-system/tasks/runtime-test/helpers-unit.js

```diff
@@ -79,28 +78,21 @@ function extractCssJsFileMap() {
 }
 
 /**
- * Returns the list of files imported by a JS file
+ * Checks whether a JS file imports any files included in possibleImports
  *
  * @param {string} jsFile
- * @return {!Array<string>}
+ * @param {Array<string>} possibleImports
+ * @return {boolean}
  */
-function getImports(jsFile) {
-  const imports = findImports([jsFile], {
-    flatten: true,
-    packageImports: false,
-    absoluteImports: true,
-    relativeImports: true,
+function areAnyDependencies(jsFile, possibleImports) {
+  const dependencies = dependencyTree.toList({
+    filename: jsFile,
+    directory: '/',
```

Comment:
If you set `directory` to `ROOT_DIR` here, you won't have to convert relative paths to absolute paths in `shouldRunTest()`.

Reply:
You still do, because `possibleImports` is given as relative paths.